ipatests/prci_definitions/nightly_latest_testing_selinux.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
         template: &testing-master-latest
-          name: freeipa/testing-master-f35
+          name: freeipa/testing-master-f36
```

Comment:
This box doesn't exist yet.

Reply:
@flo-renaud I told @f-trivino to not bump `testing-`, `pki-` and `389ds-` because I'm working on the "custom COPR" feature in PR-CI that can replace those boxes ("testing" is not a copr repo but I might address that via a variable as well).